CMakeLists.txt

```diff
@@ -618,9 +617,7 @@ set(COMPLEX_SRCS
   ${COMPLEX_SOURCE_DIR}/Utilities/OStreamUtilities.cpp
   ${COMPLEX_SOURCE_DIR}/Utilities/GeometryHelpers.cpp
   ${COMPLEX_SOURCE_DIR}/Utilities/ColorPresetsUtilities.cpp
-  ${COMPLEX_SOURCE_DIR}/Utilities/ImageRotationUtilities.cpp
```

Comment:
Was Image rotation utilities meant to be deleted too?

Reply:
Probably not?

Reply:
The original `ImageRotationUtilities` line was not aligned with the rest so the commit from #495 moved it to be in line with the rest.